packages/data-context/src/sources/VersionsDataSource.ts

```diff
@@ -0,0 +1,38 @@
+import execa from 'execa'
+
+interface Version {
+  id: string
+  released: string
+  version: string
+}
+
+export class VersionsDataSource {
+  /**
+   * Returns most recent versions and release date as a timestamp.
+   * [
+   *   { version: '8.7.0', released: '2021-10-25T21:38:59.983Z' },
+   *   { version: '8.6.0', released: '2021-10-11T19:40:49.036Z' },
+   *   { version: '8.5.0', released: '2021-09-27T20:09:18.543Z' },
+   *   { version: '8.4.1', released: '2021-09-17T21:36:14.149Z' },
+   *   { version: '8.4.0', released: '2021-09-13T20:29:16.074Z' },
+   * ]
+   */
+  async versions (n: number = 5): Promise<Version[]> {
+    const result = await execa(`npm`, [`view`, `cypress`, `time`, `--json`])
```

Comment:
Would we be better off using fetch on https://registry.npmjs.org/cypress and get the output?
This opening of a new process feels brittle.

@tgriesser what do you think?

Reply:
The mocks were out of date and we don't need to show n number of versions... I'm actually okay with this implementation though.